models/locale.php

```diff
@@ -19,19 +19,22 @@
     $locale = $_GET['locale'];
 }
 
-// get lang files status from langchecker
-$lang_files = Json::fetch(LANG_CHECKER . "?locale={$locale}&json");
+// Get lang files status from langchecker
+$json_data = new Json();
+$json_data->setURI(LANG_CHECKER . "?locale={$locale}&json");
+$lang_files = $json_data->fetchContent();
```

Comment:
Interesting. The downside is that I'm forced to create a new object if I need to fetch multiple feeds in the same view, while now I just create one instance in the file, then update the URI and fetch.

Should I switch anyway to multiple instances?